### DISCUSSION TITLE
Query raw in table user strapi

### DISCUSSION THREAD

**Discussion by KhanhKitin** [strapi/strapi#6247] [Legacy Topics]
findCard: async function() {
    var rawSql = `SELECT * FROM user-permissions_user`;
    const result = await Bookshelf.knex.raw(rawSql);
    return result;
 },

I try query but name table in db contain '-'
Can you give me a suggestion?

**Comment by lauriejim** [comment]
Hello! Can you try - await strapi.connections.default.raw(`select * from "users-permissions_user"`)

**Reply by KhanhKitin** [reply]
I tried and failed

` var rawSql = `SELECT * FROM "users-permissions_users"`;
    const result = await Bookshelf.knex.raw(rawSql);
    return result;`

error Error: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '"users-permissions_users"' at line 1

**Reply by robertpiosik** [reply]
I believe using strapi global variable mentioned by Jim above can save you from getting errors.

**Reply by KhanhKitin** [reply]
> I believe using strapi global variable mentioned by Jim above can save you from getting errors.
 I tried and failed
 error C:\Users\Quoc_Khanh\Desktop\Strapi\admin-project\api\card\services\card.js:20
    const result = await strapi.connections.default.raw(select * from "users-permissions_user");
                                                                 ^^^^

SyntaxError: missing ) after argument list
    at new Script (vm.js:79:7)
    at createScript (vm.js:251:10)
    at Object.runInThisContext (vm.js:303:10)
    at Module._compile (internal/modules/cjs/loader.js:657:28)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
error Command failed with exit code 1.

**Reply by robertpiosik** [reply]
Well, you have got a syntax error.

**Reply by KhanhKitin** [reply]
> Well, you have got a syntax error.
Can you show syntax errorr for me?
